gateway/1.8/modules/ROOT/pages/_partials/task-config-readiness-liveness.adoc

```diff
@@ -0,0 +1,187 @@
+//tag::config-readiness-liveness-page-body[]
+
+Flex Gateway supports two methods for ensuring reliability and efficiency.
+
+* Readiness probes test whether a gateway instance is configured correctly and ready for incoming traffic. This ensures that end-users are not affected by unstable or partially started services. For information, refer to <<configure-a-readiness-probe>>.
+
+* Liveness probes test whether a gateway instance is operational. This minimizes downtime due to crashes or other failures. For information, refer to <<configure-a-liveness-probe>>.
+
+[[configure-a-readiness-probe]]
+== Configure a Readiness Probe
+
+To test whether a Flex Gateway instance is ready for incoming traffic, you can use the following CLI command:
+
+[source,ssh]
+----
+flexctl probe --check=readiness
+----
+
+If the Flex Gateway instance is ready to handle incoming traffic, the command returns an exit code of `0`. Otherwise the command returns an exit code of `1`.
+
+When Flex Gateway is started, its readiness state is `false`. The state changes to `true` after the following conditions are met:
+
+* API instances are deployed (at least one API must be configured)
+* Policies are initialized
+
+The readiness state is not impacted when you apply configuration changes to a running gateway instance. This enables error-free scaling.
```

Comment:
```suggestion
The readiness state is not impacted when you apply configuration changes to a running gateway instance.
```

This is not because of error-free scaling, but rather that we don't want running flex gateways to stop handling traffic. If we would fail the readiness probe during configuration updates, then there could be some small times where no flex gateway is capable of responding to requests because they are all updating.